web/src/redux/slices/TestDefinition.slice.ts

```diff
@@ -109,7 +69,23 @@ const testDefinitionSlice = createSlice<
         return def;
       });
     },
+    revertDefinition(state, {payload: {originalSelector, selector}}) {
+      const initialDefinition = state.initialDefinitionList.find(
+        definition => definition.originalSelector === originalSelector
+      );
+
+      state.definitionList = initialDefinition
```

Comment:
We had a bug in this action, described in this video:

https://www.loom.com/share/6f8294b2c00748cd82f946c21b634e81